### DISCUSSION TITLE
"TLS 1.2 or better" required by Let's Encrypt

### DISCUSSION THREAD

**Discussion by oseiberts11** [go-acme/lego#1634] [Q&A] [answered]
From https://community.letsencrypt.org/t/rejecting-tls-1-0-1-1-for-inbound-acme-connections/176107
> ### Rejecting TLS 1.0 / 1.1 for inbound ACME connections
> 
> On September 15, 2022, we will require TLS 1.2 or better for all ACME API connections. This is in > addition to the requirement we recently announced for TLS 1.2 or better during validations (that is, > for outbound connections from our servers).
> 
> We estimate this change will affect 0.008% of certificate issuances. We'll send out emails to > affected users who have provided an email address.
> 
> If you have questions about these changes, please post them in this forum thread 18 to get support > from the Let’s Encrypt community and staff.

I presume that this is not a problem for Lego, but can somebody confirm that explicitly? Is there some minimum version required for this? We are currently using the container "goacme/lego:v4.5.2".

**Comment by ldez** [comment, marked as answer]
> I presume that this is not a problem for Lego, but can somebody confirm that explicitly?

Yes, it's not a problem for lego.

> Is there some minimum version required for this?

Not really, you can use all the lego versions since v2.
But as for all software, it's better to use the most recent version.

> We are currently using the container "goacme/lego:v4.5.2".

I recommend, for global security reasons, using the most recent version of lego.

**Reply by oseiberts11** [reply]
Thanks!
